docs/user/openstack/external_load_balancer.md

```diff
@@ -0,0 +1,225 @@
+            ip: 192.168.10.20
+      - name: ingress_http
+        vips:
+          - 192.168.10.7
+        min_backends: 1
+        healthcheck: "httpchk GET /healthz/ready HTTP/1.0"
+        frontend_port: 80
+        haproxy_monitor_port: 8082
+        balance: roundrobin
+        backend_opts: "check check-ssl port 1936 inter 1s fall 2 rise 3 verify none"
+        backend_port: 80
+        backend_hosts: *lb_hosts
+      - name: ingress_https
+        vips:
+          - 192.168.10.7
+        min_backends: 1
+        healthcheck: "httpchk GET /healthz/ready HTTP/1.0"
+        frontend_port: 443
+        haproxy_monitor_port: 8083
+        balance: roundrobin
+        backend_opts: "check check-ssl port 1936 inter 1s fall 2 rise 3 verify none"
+        backend_port: 443
+        backend_hosts: *lb_hosts
+      - name: mcs
+        vips:
+          - 192.168.10.5
+        min_backends: 1
+        frontend_port: 22623
+        haproxy_monitor_port: 8084
+        balance: roundrobin
+        backend_opts: "check check-ssl inter 5s fall 2 rise 3 verify none"
+        backend_port: 22623
+        backend_hosts: *lb1_hosts
+```
+
+**NOTE:** Note that this configuration doesn't cover the BGP capabilities of the Ansible role and it's suggested to read the role manual to learn more about it.
+
+6. Run Ansible to deploy the load balancer:
+
+```bash
+ansible-playbook -i inventory.yaml -e "@vars.yaml" playbook.yaml
+```
+
+7. To verify that your load balancer has been deployed, you can SSH into it and open `/etc/haproxy/haproxy.cfg`. It should be populated for the different backends.
+
+
+## Deploy OpenShift
+
+Now that your load-balancer(s) are ready, you can deploy OpenShift.
+Here is an example of an `install-config.yaml`:
+
+```yaml
+apiVersion: v1
+baseDomain: mydomain.test 
+compute:
+- name: worker
+  platform:
+    openstack:
+      type: m1.xlarge
+  replicas: 3
+controlPlane:
+  name: master
+  platform:
+    openstack:
+      type: m1.xlarge
+  replicas: 3
+metadata:
+  name: mycluster
+networking:
+  clusterNetwork:
+  - cidr: 10.128.0.0/14
+    hostPrefix: 23
+  machineNetwork:
+  - cidr: 192.168.10.0/24
+platform:
+  openstack:
+    cloud: mycloud
+    machinesSubnet: 8586bf1a-cc3c-4d40-bdf6-c243decc603a
+    apiVIPs:
+    - 192.168.10.5
+    ingressVIPs:
+    - 192.168.10.7
+    loadBalancer:
+      type: UserManaged
+```
+
+* `loadBalancer` is a new stanza created in OCP 4.13. The default type is `OpenShiftManagedDefault` (which will deploy HAproxy and Keepalived in OCP, known as the OpenShift managed load balancer). Setting it to `UserManaged` will allow an user managed load balancer to replace the OpenShift managed one.
+* `machinesSubnet` is the subnet ID where both the OpenShift cluster and the user managed load balancer are deployed.
+* `TechPreviewNoUpgrade` featureSet has to be enabled in OCP 4.13, since the feature was on TechPreview. To do that, just add `featureSet: TechPreviewNoUpgrade` into the install-config.yaml.
```

Comment:
"was on TechPreview" sounds weird. Can we just say "is a Technology Preview feature in this version". I assume we'll update this when it's GA?

Also, format `install-config.yaml` as a literal

Reply:
Or alternatively:

In OCP 4.13 the feature had to be enabled as Technology Preview. This can be done by adding `featureSet: TechPreviewNoUpgrade` into the `install-config.yaml`.

And link to https://github.com/openshift/installer/blob/master/docs/user/customization.md#platform-customization